plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestingIcebergPlugin.java

```diff
@@ -14,25 +14,34 @@
 package io.trino.plugin.iceberg;
 
 import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
 import io.trino.plugin.hive.metastore.HiveMetastore;
 import io.trino.spi.connector.ConnectorFactory;
 
 import java.util.List;
 import java.util.Optional;
 
 import static com.google.common.base.Verify.verify;
+import static com.google.inject.util.Modules.EMPTY_MODULE;
 import static java.util.Objects.requireNonNull;
 
 public class TestingIcebergPlugin
         extends IcebergPlugin
 {
     private final Optional<HiveMetastore> metastore;
     private final Optional<FileIoProvider> fileIoProvider;
+    private final Optional<Module> module;
```

Comment:
EMPTY_MODULE instead of Optional ?

Reply:
Ah sorry, wasn't sure on the plugin in side because of potentially more constructors to change and the trade off there. Changing now.